examples/substrate/substrate_config.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+input_files = ["flipper.sol"]   # Files to be compiled. You can define multiple files as : input_files = ["file1", "file2", ..]
+contracts = ["flipper"] # Contracts to include from the compiled files
+import_path = []   
+import_map = {}   # Maps to import. Define as  import_map = {map = "path/to/map1", map2 = "path/to/map2"}
+
+
+[target]
+name = "substrate"  # Valid targets are "solana" and "substrate"
+address_length = 32
+value_length = 16
+
+
+[debug-features]
+prints = true   # Log debug prints to the environment.
+log-runtime-errors = true   # Log runtime errors to the environment.
+generate-debug-info = false  # Add debug info to the generated llvm IR.
+
+[optimizations]
+dead-storage = true
+constant-folding = true
+strength-reduce = true
+vector-to-slice = true
+common-subexpression-elimination = true
+llvm-IR-optimization-level = "default"  # Set llvm optimizer level. Valid options are "none", "less", "default", "aggressive"
```

Comment:
This does not apply to llvm-IR only, also to target specific machine code. This should be called `llvm-optimization-level`